src/internal/UseResizeObserver.jsx

```diff
@@ -0,0 +1,23 @@
+import useResizeObserver from 'use-resize-observer';
+import warning from 'warning';
+
+import { browserSupports } from '../utils/componentUtilityFunctions';
+
+export const useResize = resizeRef => {
+  if (!browserSupports('ResizeObserver')) {
+    warning(
+      false,
+      'You have set hasOverflow to true, but the current browser does not support ResizeObserver. You will need to include a ResizeObserver polyfill for hasOverflow to function properly.'
```

Comment:
I think you need to modify this warning text to fit the generic purpose of this HOC

Reply:
pushing a fix up right now